### PR TITLE
Add bash tips

### PR DESCRIPTION
**Pull request by TomOnTime**
* Suggest some friendly env/aliases for your .bashrc

### CODE REVIEW

**Comment by devopsdays-service** [comment]
@TomOnTime, thanks! Based upon the content in this PR, we need a review from @mattstratton and @bridgetkromhout

File: utilities/README.md [hunk 1]
```diff
@@ -133,3 +133,36 @@ speakerdeck = ""
...
+Set `$DOD_YEAR` and `$DOD_CITY` and the scripts won't ask
+for the year and city every time.
+
+Here are some useful bash exports/aliases. Add them to your `.bashrc`:
+
+```
+# devopsdays
+export DOD_YEAR="2018"            # your year
+export DOD_CITY="new-york-city"   # your city
+export DOD_DIR=~/git/devopsdays-web   # location of Git files
```
Comment (mattstratton) [inline utilities/README.md]: Silly little nit...can you make this `DODPATH`? The cli tool uses the env variable and it will make this portable.

Comment (TomOnTime) [reply utilities/README.md]: Glad to!

**Comment by TomOnTime** [comment]
PTAL

**Comment by TomOnTime** [comment]
Ready for review.

**Comment by bridgetkromhout** [comment]
LGTM.

**Pull request merged**